iceoryx_hoofs/test/integrationtests/test_resizeable_lockfree_queue_stresstest.cpp

```diff
@@ -399,7 +399,7 @@ TYPED_TEST_SUITE(ResizeableLockFreeQueueStressTest, TestConfigs);
 
 ///@brief Tests concurrent operation of multiple producers and consumers
 ///       with respect to completeness of the data, i.e. nothing is lost.
-TYPED_TEST(ResizeableLockFreeQueueStressTest, multiProducerMultiConsumerCompleteness)
+TYPED_TEST(ResizeableLockFreeQueueStressTest, DISABLED_multiProducerMultiConsumerCompleteness)
```

Comment:
I also created an issue how to handle those stress tests in a clean fashion: https://github.com/eclipse-iceoryx/iceoryx/issues/1274